engine/src/test/java/net/jqwik/api/BaseDomainContexts.java

```diff
@@ -0,0 +1,15 @@
+package net.jqwik.api;
+
+import net.jqwik.api.domains.AbstractDomainContextBase;
+
+abstract class BaseDomainContexts {
+
+	Integer[] specificNumbers;
```

Comment:
I should note that I put this in there to ensure that the inner class is non-`static`.